### ISSUE TITLE
0.9.3 broken/missing mods

### ISSUE THREAD

**Issue by gardebreak** [Mord3rca/gamma-launcher#178]
**Distro:** up-to-date Arch

**Setup procedure:**

1. Perform a fresh full install using `gamma-launcher`

```
gamma-launcher full-install --anomaly /data/games/stalker-anomaly-test --gamma /data/games/stalker-gamma-test --cache-directory /data/games/stalker-cache-test
```

2. Set up a fresh wineprefix

```
WINEPREFIX=~/.local/share/wineprefixes/stalker-gamma-test winetricks cmd d3dcompiler_47 d3dx10 d3dx11_43 d3dx9 dx8vb dxvk quartz vcrun2019
```

3. Run Anomaly via Lutris using Wine 9.20 (system Wine) and the wineprefix previously set up. Exit the game after the main menu loads.

4. Run GAMMA via Lutris using Wine 9.20 and the same wineprefix. MO2 opens with 506 mods active (should be 507.)

At this point I'm seeing different behavior from what I was seeing yesterday (possible something changed in the GAMMA back end I guess?)

Yesterday, launching `Anomaly (DX11)` from MO2 would crash with a message in the Anomaly log about some mod files conflicting/needing an override.

Today, GAMMA does launch successfully and the main menu loads. However there are still issues with a few mods in MO2:

These three mods are flagged with the red X in MO2 (missing data.) Only the third one is actually enabled.

* 142- Fixed Crafting with Multi-Use Items WPO Patch - thisisntmysteamid
* 242- Gardener of the Zone Textures - YuriVernadsky
* 376- Barbed Wire Shadow Fix - VodkaPivin

Copying over those three mod folders from my Windows GAMMA install gets rid of the red X's.

The missing mod is `378- FDDA Enhanced Animations - lizzardman`. I found this by diffing the MO2 G.A.M.M.A. profile `modlist.txt` files from the Linux and Windows installs. I copied over that mod from the Windows install, placed it in the correct spot in the load order, and enabled it resulting in the correct count of 507 mods.

**Comment by Mord3rca** [comment]
For the 3rd red X mod:
> https://www.moddb.com/addons/start/271165	Main (Required):REPACK_Barbed_Wire (Recommended)	 - VodkaPivin	 Barbed Wire Shadow Fix	https://www.moddb.com/mods/stalker-anomaly/addons/barbwire-shadow-fix
 
 File: Camo_net_Fix1.02.zip
 Installation failed since *Main (Required)/* & *REPACK_Barbed_Wire (Recommended)/* do not exist.
 
```
Extracting <path/to/download>/Camo_net_Fix1.02.zip
    Installing Main (Required) -> </path/to/gamma/mods>/376- Barbed Wire Shadow Fix - VodkaPivin
    WARNING: Main (Required) does not exist
    Installing REPACK_Barbed_Wire (Recommended) -> </path/to/gamma/mods>/376- Barbed Wire Shadow Fix - VodkaPivin
    WARNING: REPACK_Barbed_Wire (Recommended) does not exist
```

Mod archive was probably updated

**Comment by Mord3rca** [comment]
Now regarding `378- FDDA Enhanced Animations - lizzardman`, this mod is installed but with on top of  `371- FDDA Enhanced Animations - Mirrowel & MFS Team & Gunslinger Team`.
I'll find a better way to fix that during the week.

However, since it is installed on top of another mod .... It just works.

**Comment by Mord3rca** [comment]
It will cause some issue, the modification was made here 85b76123e219d306c33399328bfb03d310108d6b to fix missing git mods: https://github.com/Mord3rca/gamma-launcher/issues/68#issuecomment-1869179717

**Comment by Mord3rca** [comment]
Can you try #180 ? I have 508 mod active with it.

Note: I did not have a reference installation for now ... i'll create it tomorrow and `meld` the two to see if nothing fishy happened after this modification.

**Comment by gardebreak** [comment]
#180 works for me too. 508 mods (the correct amount as of right now) and GAMMA launches successfully.

**Comment by MichaelHochriegl** [comment]
Sorry for posting on this already closed issue, but I tried #180  before the merge and just now I tried the newly fetched master branch after this was merged and I still get the `no valid gamedata` (aka the red cross) for the three mods also mentioned in the first post:
1.     142- Fixed Crafting with Multi-Use Items WPO Patch - thisisntmysteamid
2.     242- Gardener of the Zone Textures - YuriVernadsky
3.     376- Barbed Wire Shadow Fix - VodkaPivin

I tried reinstalling now three times (once a complete fresh install). Am I doing anything wrong?
